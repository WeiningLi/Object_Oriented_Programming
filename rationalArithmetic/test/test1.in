create a 5/1
print a
create b 5/15
print b
+ a b
- b a
* a b
/ b a
+= a b
-= a b
u- a

