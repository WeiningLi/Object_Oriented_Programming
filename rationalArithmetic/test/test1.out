5
1/3
16/3
-14/3
5/3
1/15
16/3
5
-5
